api/apps/articles/model/schema.coffee

```diff
@@ -75,7 +75,10 @@ denormalizedArtwork = (->
   thumbnail_title: @string().allow('', null)
   thumbnail_teaser: @string().allow('', null)
   thumbnail_image: @string().allow('', null)
+  vertical: @array().items(@string()).default([])
```

Comment:
is there a chance these may turn into something more like a curation, with an object id?